charts/ratify/templates/deployment.yaml

```diff
@@ -55,7 +62,10 @@ spec:
             {{- if (lookup "v1" "Secret" .Release.Namespace "gatekeeper-webhook-server-cert") }}
             - --ca-cert-file=usr/local/tls/client-ca/ca.crt
             {{- end }}
-            - --cache-size={{ .Values.provider.cache.size }}
+            - --cache-enabled={{ .Values.provider.cache.enabled }}
+            - --cache-type={{ default "ristretto" .Values.provider.cache.type }}
```

Comment:
The cacheType corresponds to the cache provider registered implementation. So if the cache type is not 'ristretto' or 'dapr' then Ratify will throw an error similar to how verifiers/stores/policy handles it. However, it is totally valid for a cache-type to be any value. Since this value is associated with value User configures for the statestore Component CR